MAUI/Kanban-Board/Column.md

```diff
@@ -227,15 +227,15 @@ The following output demonstrates the above example code.
 
 {% highlight xaml %}
 
-    <kanban:KanbanColumn x:Name="todoColumn" Title="To Do" MinimumLimit="2" MaximumLimit="1">
```

Comment:
@EzhilarasanElangovan31 in line 221, update along with its type in APIs like below
<img width="542" alt="image" src="https://github.com/user-attachments/assets/00ed4b47-ec21-4988-b2ec-a67d2148ed32">

Reply:
Updated